clang/test/CAS/path-independent-cas-outputs.c

```diff
@@ -58,15 +58,12 @@
 // RUN: diff %t/a/t1.d %t/b/t2.d
 // RUN: diff %t/t.d %t/a/t1.d
 
-// Baseline to check we got expected output.
```

Comment:
why is this related to the change?

Reply:
The PCH emitted with include tree is not binary compatible with non-cached PCH, because it removes the header search paths that get stored in the PCH (in `HEADER_SEARCH_PATHS` record).